src/dat/sys/routes.clj

```diff
@@ -0,0 +1,44 @@
+(ns dat.sys.routes
```

Comment:
Seems like a good separation of concerns. My main reservation is that I was planning on using bidi by default for actual application routing. And it would be nice to have that be a `routes.cljc` that's part of the standard set up of things where you share those routes between client and server.

But maybe those things could still live here? We could maybe make it wild-card so that if it doesn't match any of the top level compojure routes, it can check to see if it's a valid client side route, and render that page instead (pre-compiled?).

In any case, this is probably fine for now if it works, but we should think about this question a little bit. Thoughts?